sql/CMakeLists.txt

```diff
@@ -427,3 +427,9 @@ INSTALL(DIRECTORY . DESTINATION ${INSTALL_INCLUDEDIR}/private COMPONENT Developm
   FILES_MATCHING PATTERN "*.h"
   PATTERN share EXCLUDE
   PATTERN CMakeFiles EXCLUDE)
+
+ADD_CUSTOM_COMMAND(TARGET
+  mysqld
+  POST_BUILD
+  COMMAND if [ -f "${CMAKE_BINARY_DIR}/compile.warnings" ]; then cat "${CMAKE_BINARY_DIR}/compile.warnings" \; fi
+  DEPENDS mysqld)
```

Comment:
Modification of several `CMakeLists.txt` is highly unwanted. P.1 of task is not done.